addons/account/models/company.py

```diff
@@ -720,22 +720,23 @@ def get_unaffected_earnings_account(self):
         """ Returns the unaffected earnings account for this company, creating one
         if none has yet been defined.
         """
+        AA = self.env['account.account'].with_company(self)
```

Comment:
```suggestion
        AccountAccount = self.env['account.account'].with_company(self)
```

I would have preferred `self = self.with_company(self)` but then ruff would complain :shrug: